engine/admin/live-restore.md

```diff
@@ -71,4 +71,7 @@ You can modify the kernel's buffer size by changing `/proc/sys/fs/pipe-max-size`
 
 The live restore option is not compatible with Docker Engine swarm mode. When
 the Docker Engine runs in swarm mode, the orchestration feature manages tasks
-and keeps containers running according to a service specification.
\ No newline at end of file
+and keeps containers running according to a service specification.
+
+ **Note**: The live restore option is only supported on Linux.
```

Comment:
Oh, you must hate me by now :smile:, we standardize on using a "quote" formatting for notes, so that they are clearly recognizable as a "note". Basically, just add a `>` at the start;

```Markdown
> **Note**: The live restore option is only supported on Linux.
```

Then it will look like this in the rendered documentation;

<img width="629" alt="screen shot 2016-11-15 at 14 54 43" src="https://cloud.githubusercontent.com/assets/1804568/20308309/92105c16-ab43-11e6-9a58-408ffceebf44.png">